examples/c/ex_all.c

```diff
@@ -1228,8 +1228,8 @@ main(int argc, char *argv[])
     /*
      * Configure eviction to begin at 90% full, and run until the cache is only 75% dirty.
      */
-    error_check(
-      wiredtiger_open(home, NULL, "create,eviction_trigger=90,eviction_dirty_target=75", &conn));
```

Comment:
Good catch: This is leftover from the days when dirty trigger/target were 95/80 as compared to 20/5 now. I suppose no harm in leaving examples to show values that made more sense in the past.